Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -106,7 +110,7 @@ class _SearchLoader<UnusedA, UnusedB>: Loader<Cursor<Site>, SearchViewController
             // so make sure the result includes at least one ".".
             let matchedDomain: String = domainWithDotPrefix.substring(from: domainWithDotPrefix.index(range.lowerBound, offsetBy: 1))
             if matchedDomain.contains(".") {
-                return matchedDomain + "/"
```

Comment:
Do we need to actually do this to fix this patch since we're already removing the trailing slash from `prePathURL`? Most browsers, including Firefox Desktop, add the trailing slash when autocompleting domains.